pkg/controller/pxcbackup/controller.go

```diff
@@ -621,3 +633,30 @@ func setControllerReference(cr *api.PerconaXtraDBClusterBackup, obj metav1.Objec
 	obj.SetOwnerReferences(append(obj.GetOwnerReferences(), ownerRef))
 	return nil
 }
+
+func (r *ReconcilePerconaXtraDBClusterBackup) isOtherBackupRunning(ctx context.Context, cr *api.PerconaXtraDBClusterBackup) (bool, error) {
```

Comment:
You could rename this as just `isBackupRunning`, it is implied that it is about some other backup from the caller.

Reply:
I don't agree. this function checks if a backup other than current CR is running. I think this name makes more sense.

Reply:
Yes, you're right.